tests/test_feedexport.py

```diff
@@ -601,7 +616,7 @@ def test_export_no_items_not_store_empty(self):
                 },
             }
             data = yield self.exported_no_data(settings)
-            self.assertEqual(data[fmt], b'')
+            self.assertEqual(b'', data[fmt])
```

Comment:
Was there a reason for this change? :slightly_smiling_face:

Reply:
I saw such tox error output "expected != got" or something like this, so I supposed that first value was considered as expected, second - got. For some reason, I can't reproduce such situation now.